src/content/concepts/module-federation.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Module Federation
+contributors:
+  - sokra
+  - chenxsan
+related:
+  - title: "Webpack 5 Module Federation: A game-changer in JavaScript architecture"
+    url: https://medium.com/swlh/webpack-5-module-federation-a-game-changer-to-javascript-architecture-bcdd30e02669
+---
+
+## Motivation
+
+Multiple separate builds should form a single application. These separate builds should not have build dependencies between each other, so they can be developed and deployed individually.
+
+This is often known as Micro-Frontends, but is not limited to that.
+
+## Low-level concepts
+
+We distinguish between local and remote modules. Local modules are normal modules which are part of the current build. Remote modules are modules that are not part of the current build and loaded from a so called container at runtime.
+
+Loading remote modules is considered as async operation. When using a remote module these async operation will be placed in the next chunk loading operation(s) that is between the remote module and the entrypoint. It's not possible to use a remote module without a chunk loading operation.
```

Comment:
```suggestion
Loading remote modules is considered an asynchronous operation. When using a remote module, these asynchronous operations, will be placed in the next chunks loading operation(s). That chunk is between the remote module and the entrypoint. It's not possible to use a remote module without a chunk loading operation.
```

Reply:
It's the **next chunks loading operation(s)** that is between the remote module and entrypoint, or **chunk** itself?